src/style.css

```diff
@@ -1 +1,107 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+body {
+  padding-top: 20px;
+  margin: 0 8;
+
+  --viewport-width: calc(100vw - 16px);
+}
+
+.big-search-bar {
+  width: var(--viewport-width);
+  height: 70px;
+  margin-bottom: 20px;
+}
+
+.big-search-bar__input {
+  width: 100%;
+  box-sizing: border-box;
+  height: 70px;
+
+  padding-left: 62px;
+  background-image: url(images/Search.svg);
+  background-size: 19px 19px;
+  background-repeat: no-repeat;
+  background-position: 26px center;
+  border: 1px solid #E1E7ED;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3D4E611A;
+  font-family: Avenir, sans-serif;
+  color: #3D4E61;
+  font-size: 16px;
+}
+
+.big-search-bar__input:hover {
+  box-shadow: 0 3px 8px 0 #3D4E6133;
+}
+
+.big-search-bar__input:focus {
+  box-shadow: 0 3px 8px 0 #3D4E6133;
+  outline: none;
+  background: url(images/Search.svg) 26px center no-repeat,
+  linear-gradient(180deg, #FFF 0%, #F6F6F7 100%);
+  background-size: 19px 19px;
+  text-shadow: 0 4px 4px 0 #00000040;
+  color: #3D4E61;
+}
+
+.small-search-bar {
+  width: var(--viewport-width);
+  height: 42px;
+}
+
+.small-search-bar__input {
+  width: 100%;
+  box-sizing: border-box;
+  height: 42px;
+
+  padding-left: 33px;
+  background-image: url(images/Search.svg);
+  background-size: 11px 11px;
+  background-repeat: no-repeat;
+  background-position: 13px center;
+  border: 1px solid #E1E7ED;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3D4E611A;
+  font-family: Avenir, sans-serif;
+  font-weight: 400;
+  color: #3D4E61;
+  font-size: 14px;
```

Comment:
Do not repeat your self. Use class .search-bar that will have all properties, and only usign modifiers - change required properties